backstory/pom.xml

```diff
@@ -35,7 +35,34 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.9.5</version>
+       <scope>test</scope>
+    </dependency>
```

Comment:
Are these dependencies related to this PR?

Reply:
I added dependencies I thought would be useful for the project later on, as I was setting up the file structure. As such, I added mockito (I also later updated the version of mockito), junit and the java client I was using for Perspective. They aren't related to this PR though. @ykjchen I can take them out if you want.

Reply:
I took them out.

Reply:
Done